### DISCUSSION TITLE
How to get my PR merged?

### DISCUSSION THREAD

**Discussion by InvisibleFunction** [beetbox/beets#5504] [General]
I've had this PR that's just been hanging out since March. How can get this merged? Can someone help me write some tests? Will that help?

https://github.com/beetbox/beets/pull/5149

**Comment by snejus** [comment]
There's nothing wrong with the change you've made in there - I think this has more to do with maintainers lacking capacity for reviews 😢

**Reply by InvisibleFunction** [reply]
Thanks for the help!
